src/components/ec-file-list/ec-file-list.spec.js

```diff
@@ -0,0 +1,46 @@
+import { mount, createLocalVue } from '@vue/test-utils';
+import EcFileList from './ec-file-list.vue';
+
+describe('EcFileList', () => {
+  function mountFileList(props, mountOpts) {
+    const localVue = createLocalVue();
+
+    return mount(EcFileList, {
+      localVue,
+      propsData: { ...props },
+      ...mountOpts,
+    });
+  }
+
+  const items = [
+    { name: 'My invoice.pdf' },
+    { name: 'This is a very long file name that is for testing the ellipses.pdf' },
+    { name: 'DOC_123123_2423490802348_12312323.pdf' },
+    { name: 'number_rates_trades.xscl' },
+    { name: 'untitled.pdf' },
+  ];
+  describe('EcFileList', () => {
```

Comment:
Please add an empty line before `describe()` block and between each `it()`. It's just nicer to look at. There is no eslint to enforce that unfortunately, so we have to do that manually.